resources/public/api/conf/1.0/endpoints/charitableGiving_retrieve.raml

```diff
@@ -0,0 +1,27 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotSupported
```

Comment:
In this description you can get rid of `That is,` to match the spec description

Reply:
Updated